staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -766,6 +766,11 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdownBasedOnPodPriority
 	// +optional
 	ShutdownGracePeriodByPodPriority []ShutdownGracePeriodByPodPriority `json:"shutdownGracePeriodByPodPriority,omitempty"`
+	// CrashLoopBackOff contains config to modify node-level parameters for
+	// container restart behavior
+	// +featureGate=KubeletCrashLoopBackOffMax
+	// +optional
```

Comment:
This doesn't need to be a pointer, I think, or marked as optional - the struct is OK, even if none of the fields in it are set.

Reply:
docs, mostly

Reply:
Left in the `+optional` part still, see also [discussion here](https://github.com/kubernetes/kubernetes/pull/128374/files#r1830430379)